5.traits.md

```diff
@@ -153,14 +153,14 @@ spec:
       instanceName: web-front-end
       parameterValues:
       traits:
-        - name: ManualScaler
+        - name: manual-scaler.core.oam.dev/v1alpha1
```

Comment:
This Trait name (i.e. this example from Rudr) does work but is essentially NOT right (i.e. Trait type is missing).

I'd suggest use right format here (e.g. update the spec and this example to include both Trait name and type), and send another PR to fix Rudr at the same time.

/cc @wonderflow

Reply:
yes